reviews/2022-metal3-io-annual.md

```diff
@@ -0,0 +1,264 @@
+throughout the year.
+
+### Downloads per repository:
+We have fairly high number of downloads of our release manifests and
+artifacts. We showcase two examples below.
+
+[Cluster API Provider Metal3](https://hanadigital.github.io/grev/?user=metal3-io&repo=cluster-api-provider-metal3)
+
+[IP Address Manager](https://hanadigital.github.io/grev/?user=metal3-io&repo=ip-address-manager)
+
+In addition, we have a very high number of downloads of our container images
+from [quay.io](https://quay.io/). Since the download statistics is not
+publicly visible, we are not adding any link.
+
+As an example of our traction the aggregated number of container image
+downloads from the Metal3's container repository from `10-26-2022 to
+11-2-2022` was on `average 12193` and the peak was `15755 during a single day`.
+
+![](https://raw.githubusercontent.com/Nordix/metal3-clusterapi-docs/master/2022-metal3-io-annual-review/2022-metal3-io-container-downloads.png)
+
+
+## Current maintainers
+
+The following table contains both the project's approvers and reviewers combined.
+
+| Reviewer/Approver  | GitHub ID      | Affiliation                    |
+| ------------------ | -------------- | ------------------------------ |
+| Andrea Fasano      | andfasano      | Red Hat                        |
+| Bob Fournier       | bfournie       | Red Hat                        |
+| Derek Higgins      | derekhiggins   | Red Hat                        |
+| Dmitry Tantsur     | dtantsur       | Red Hat                        |
+| Riccardo Pittau    | elfosardo      | Red Hat                        |
+| Honza Pokorny      | honza          | Red Hat                        |
+| Himanshu Roy       | hroyrh         | Red Hat                        |
+| Zane Bitter        | zaneb          | Red Hat                        |
+| Iury Gregory       | iurygregory    | Red Hat                        |
+| Kashif Khan        | kashifest      | Ericsson Software Technology   |
+| Furkat Gofurov     | furkatgofurov7 | Ericsson Software Technology   |
+| Mohammed Boukhalfa | mboukhalfa     | Ericsson Software Technology   |
+| Lennart Jern       | lentzi90       | Ericsson Software Technology   |
+| Adil Ghaffar       | adilGhaffarDev | Ericsson Software Technology   |
+| Moshiur Rahman     | smoshiur1237   | Ericsson Software Technology   |
+| Sunnatillo Samadov | Sunnatillo     | Ericsson Software Technology   |
+| Adam Rozman        | Rozzii         | Ericsson Software Technology   |
+
+Complete list of approvers across different repositories can be
+found [here](https://github.com/metal3-io/metal3-docs/blob/master/maintainers/ALL-OWNERS).
+We have introduced the option of moving people to emeritus approvers and
+reviewers list for people who became less active in the review/maintenance
+process.
+
+## Adoption
+
+The project's adopters list can be found [here](https://github.com/metal3-io/metal3-docs/blob/main/ADOPTERS.md).
+
+**Adopters:**
```

Comment:
This seems duplicate as there is header just above, and its followed by equally strong line.

Reply:
Thanks, done.